google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/CreateTableRequest.java

```diff
@@ -103,13 +87,13 @@ public boolean equals(Object o) {
       return false;
     }
     CreateTableRequest that = (CreateTableRequest) o;
-    return Objects.equal(createTableRequest, that.createTableRequest)
+    return Objects.equal(requestBuilder, that.requestBuilder)
```

Comment:
This is unrelated to this PR, but this looks broken. I don't think proto builders implement value equality checks. In a separate PR, can you add a unit test for equality and if its broken, fix it, by comparing requestBuilder.build() to that.requestBuilder.build(). Or something more clever

Reply:
Have created #5079 to address fix.